.github/workflows/portlabel.yml

```diff
@@ -0,0 +1,16 @@
+name: Port Request Labeler
+
+on:
+  pull_request:
+    branches:
+      - release/**
+
+jobs:
+  release_labeler:
+    name: Release labeler
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aaomidi/AutoLabel@master
```

Comment:
Can we have this checked in somewhere in the repo instead?

Reply:
I don't know where in the project to put it so I don't break other scripts as we use globs a lot.

Reply:
What do you mean? you can have a sub-project under the build direction for github actions

Reply:
Well they build completely differently. Do we want to keep the dist folder inside ADS's code base? 

https://github.com/aaomidi/AutoLabel/tree/master/dist
https://github.com/aaomidi/AutoMerge/tree/master/dist

I really don't think these files should end up in ADS as they're basically compiled & webpacked.

I also don't want to spend time re-compiling this everytime we use it.

Reply:
I think this fine to check in compiled files for things like this, its easier to track these sort of items when they are checked in close to where they are used.